src/esp/core/ManagedContainerBase.cpp

```diff
@@ -159,9 +161,9 @@ bool ManagedContainerBase::verifyLoadDocument(const std::string& filename,
     return true;
   } else {
     // by here always fail
-    LOG(ERROR) << objectType_
-               << "ManagedContainerBase::verifyLoadDocument : File " << filename
-               << " does not exist";
+    !Cr::Utility::Error{} << objectType_
```

Comment:
FYI there are slightly shorter `Mn::Debug`, `Mn:Error` aliases to save you typing.

(These are not really visible in the docs, I'll fix that.)